app/data_model/list_store.py

```diff
@@ -40,6 +40,14 @@ def serialise(self):
     def __repr__(self):
         return f"<ListModel name={self.name} items={self.items}, primary_person={self.primary_person}>"
 
+    @property
+    def first(self):
+        try:
+            return self.items[0]
+        except IndexError:
+            logger.error("unable to access first item in list, list is empty")
+            raise IndexError
```

Comment:
```suggestion
            raise
```

Reply:
From the docs:
`If no expressions are present, raise re-raises the last exception that was active in the current scope. If no exception is active in the current scope, a RuntimeError exception is raised indicating that this is an error.`

https://docs.python.org/3/reference/simple_stmts.html#raise